cli/azd/pkg/input/console.go

```diff
@@ -149,24 +151,43 @@ func (c *AskerConsole) MessageUxItem(ctx context.Context, item ux.UxItem) {
 	}
 }
 
+func (c *AskerConsole) spinnerText(title, charset string) string {
```

Comment:
Is there a way we can emulate a console and provide test coverage here?

Reply:
I don't think there's an easy way for us to do this, FWIW.  I think I'm okay with not having coverage here.

Reply:
at least I added some tests to cover the truncating method to be as expected.